From 81c1781b3d148fb3853af8cc72bb403846a2b7ed Mon Sep 17 00:00:00 2001
From: frankheckenbach <fjf@gmx.de>
Date: Fri, 22 Apr 2016 14:21:30 +0200
Subject: [PATCH] tmp.mount.hm4: After swap.target (#3087)

fix issue #2930
Cherry-picked from: a11fe93e04e775c3ce2ace92be761d5ff9fce2d9
Resolves: #1298355
---
 units/tmp.mount | 1 +
 1 file changed, 1 insertion(+)

diff --git a/units/tmp.mount b/units/tmp.mount
index af0cf4a55..8c53a8705 100644
--- a/units/tmp.mount
+++ b/units/tmp.mount
@@ -13,6 +13,7 @@ ConditionPathIsSymbolicLink=!/tmp
 DefaultDependencies=no
 Conflicts=umount.target
 Before=local-fs.target umount.target
+After=swap.target
 
 [Mount]
 What=tmpfs
